src/test/java/uk/gov/hmcts/reform/judicialapi/elinks/service/impl/ElinksPeopleServiceImplTest.java

```diff
@@ -722,7 +721,7 @@ void loadPeopleForTribunalsWithParentIdEmptyWhiteSpaces() throws JsonProcessingE
                 .authorisationsRequests(List.of(AuthorisationsRequest.builder().jurisdiction("juristriction")
                         .ticket("lowerlevel").startDate("1991-12-19")
                         .endDate("2022-12-20").ticketCode("ticketId").build()))
-                .judiciaryRoles(Lists.newArrayList()).build();
+                .judiciaryRoles(List.of()).build();
```

Comment:
minor: emptyList() imported statically from java.util.Collections would be more readable (same applies for other tests where an empty list is defined)